doc/autoupdate.rst

```diff
@@ -32,10 +32,9 @@ itself. Should the silent update fail, the client offers a manual download.
 macOS
 ^^^^^
 
-If a new update is available, the Nextcloud client initializes a pop-up dialog
-to alert you of the update and requesting that you update to the latest
-version. Due to their use of the Sparkle frameworks, this is the default
-process for macOS applications.
+There is no automatic updater on macOS. If a new update is available,
+the Nextcloud client initializes a pop-up dialog to alert you of the
+update and requesting that you update to the latest version.
```

Comment:
@FlexW 
`the latest version.` -> `the latest version` **manually**`.`

I guess this would be more precise?

Reply:
I agree